src/Jmelosegui.Mvc.Googlemap.Examples/Views/Basic/Drawing.cshtml

```diff
@@ -0,0 +1,76 @@
+﻿@using Jmelosegui.Mvc.GoogleMap
+
+@{
+    ViewBag.Title = "Drawing tools";
+    string[] libraries = { "drawing" };
+
+}
+
+<div style="height: 500px; border: solid 1px #cccccc">
+    @(Html.GoogleMap()
+        .Name("map")
+        .Height(500)
+        .Libraries(libraries)
+        .ClientEvents(events => events.OnMapLoaded("mapEvent"))
+    )
+</div>
+
+<br />
+
+@section scripts{
+    @(Html.GoogleMap().ScriptRegistrar())
```

Comment:
The `ScriptRegistrar()` method is already called within the [_LayoutExamples.cshtml](https://github.com/jmelosegui/GooglemapMvc/blob/master/src/Jmelosegui.Mvc.Googlemap.Examples/Views/Shared/_LayoutExamples.cshtml#L17). If we include it again will cause the following error:

>You have included the Google Maps API multiple times on this page. This may cause unexpected errors.